home/tests.py

```diff
@@ -423,3 +423,9 @@ def test_process_request_with_logged_out_user(self):
         request_mock = Mock(user=self.LoggedOutUserMock())
         pm.process_view(request_mock, None, None, None)
         assert request_mock.profile is None
+
+
+@pytest.mark.django_db
+def test_login(client):
+    response = client.get('/login')
+    assert response.status_code == 301
```

Comment:
I'd expect a bit more then that - for example, where are you being redirected to?